components/membership/membership-client-impl/src/main/kotlin/net/corda/membership/impl/client/MemberOpsClientImpl.kt

```diff
@@ -53,6 +54,8 @@ class MemberOpsClientImpl @Activate constructor(
 
         const val CLIENT_ID = "membership.ops.rpc"
         const val GROUP_NAME = "membership.ops.rpc"
+
+        private val clock: Clock = UTCClock()
```

Comment:
```suggestion
        private val clock = UTCClock()
```
will make it more Kotlin-style, and also remove the need for the additional `Clock` import.

Reply:
Made the suggested change at all places